src/WebJobs.Script/Workers/Http/HttpFunctionInvocationDispatcher.cs

```diff
@@ -120,12 +120,12 @@ public async void WorkerRestart(HttpWorkerRestartEvent workerRestart)
 
         private async Task DisposeAndRestartWorkerChannel(string workerId)
         {
-                _logger.LogDebug("Disposing channel for workerId: {channelId}", workerId);
-                if (_httpWorkerChannel != null)
-                {
-                    (_httpWorkerChannel as IDisposable)?.Dispose();
-                }
-                await RestartWorkerChannel(workerId);
+            _logger.LogDebug("Disposing channel for workerId: {channelId}", workerId);
```

Comment:
Why is the format name for workerId channelId?

Reply:
Guessing a refactoring leftover :/

Reply:
And I didn't actually change that... this was a spacing auto-cleanup that happened on save.